docs/governance/working-groups/security/charter.md

```diff
@@ -0,0 +1,53 @@
+# Security Working Group
+
+This [WG] defines how a threat-informed control catalog can be used to assess the security of the common cloud services defined by the CCC services taxonomy working group.
+
+## Mission
+
+The mission of the Security [WG] is to develop, maintain, and enhance a comprehensive threat-informed security control catalog specifically tailored to the CCC cloud services taxonomy. As part of the mission, the working group will explore how existing catalogs, knowledge bases and assessment frameworks can be leveraged to assess the security posture of CCC-relevant cloud services and ensure alignment with industry standards and best practices.
+
+## Approach
+
+- Maintain open and transparent communication channels for all WG activities and decisions in regard to outputs/deliverables
+
+## Responsibilities
+
+- Define and maintain a threat-informed security control catalog outlining effective measures to mitigate identified threats relative to the CCC services taxonomy.
+- Define methodologies and criteria for assessing the effectiveness of security controls, including leveraging existing security controls catalogs, threat knowledge bases, risk-based assessments, assessment formats (e.g. OSCAL), and testing procedures.
+- Explore how existing security controls catalogs, such as CIS benchmarks, NIST SP 800-53, ISO/IEC 27001, as well as threat knowledge bases, such as MITRE ATT&CK, can be utilized to assess the security posture of cloud services.
+- Collaborate with industry experts, security researchers, and cloud service providers to ensure the accuracy and relevance of the catalogs, assessment methodologies, and exploration of existing controls catalogs.
+- Regularly update and expand the catalogs, assessment methodologies, and exploration of existing control catalogs to reflect evolving threats, emerging technologies, and best practices in cloud security.
+
+### Input
+
+The input to the WG is the output from the services taxonomy WG.
+
+### Output / Deliverables
+
+This group should, at minimum, create and maintain the following items in the project GitHub repository.
+
+- Threat catalog 
+- Control catalog
+
+## Out of Scope
+
+This [WG] is not responsible for technicial work that does not pertain to the management of the CCC threat and control catalogs.
+
+## Membership
+
+The membership structure of this [WG], including roles and responsibilities, must adhere to the latest recommendations from the Community Structure [WG].
+
+Where recommendations do not yet exist on a topic, this [WG] should follow guidance from the CCC [SC].
+
+## Community Cadence
+
+A member of this [WG] should be present on for any calls scheduled by the [SC] for participation by the full CCC community, such as quarterly [SC] calls.
+
+This [WG] will use the mail group <ccc-security@lists.finos.org> for regular communications.
```

Comment:
A suggestion from @crawfordchanel: We should make a formal calendar meeting for this WG with a weekly or bi-weekly cadence. That decision should be captured here.